documentation/introduction/introduction.tex

```diff
@@ -70,6 +70,13 @@ \section{Getting started}
 
 \section{Changes from previous versions}
 
+\subsection{Changes from v1.5.0}
```

Comment:
Not required, this is about Dynawo versions

Reply:
Ok I will remove this section.